docs/guides/latest/platforms/aks.md

```diff
@@ -633,4 +633,4 @@ kubectl delete -n demo restoresession deployment-restore
 kubectl delete -n demo repository azure-repo
 kubectl delete -n demo secret azure-secret
 kubectl delete -n demo pvc --all
-```
\ No newline at end of file
+```
```

Comment:
We need a next steps section. We need to guide someone who just arrived at this page for their desired cloud:

- Links to other types of workloads
- Links to PVC
- Links to database?